src/commands/teardown.rs

```diff
@@ -91,6 +91,7 @@ impl Teardown {
                 firewall: firewall_driver.as_ref(),
                 container_id: &network_options.container_id,
                 container_name: &network_options.container_name,
+                container_dns_servers: &None,
```

Comment:
Why is this none? I understand that it is not required for teardown but it is inconsistent, just pass the nameservers along